Controller/ResourceController.php

```diff
@@ -39,11 +45,104 @@ public function __construct(
         $this->registry = $registry;
     }
 
-    public function resourceAction($repositoryName, $path)
+    public function getResourceAction($repositoryName, $path)
     {
         $repository = $this->registry->get($repositoryName);
         $resource = $repository->get('/'.$path);
 
+        return $this->createResponse($resource);
+    }
+
+    /**
+     * There are two different changes that can currently be done on a cmf resource:
+     *
+     * - move
+     * - rename
```

Comment:
At least my move PR on Puli does support renaming in the move action (as @dbu said, both are just about paths): https://github.com/puli/repository/pull/26/files#diff-14ae094154456c4c7c9fa3ddb909c559R388